Projects/Embark/Sources/Viewables/Actions/TextActionSet.swift

```diff
@@ -72,6 +98,36 @@ extension TextActionSet: Viewable {
                     }
                 }
             }
+            
+            bag += containerView.applyShadow({ (traitCollection) -> UIView.ShadowProperties in
+                UIView.ShadowProperties(
+                    opacity: 1,
+                    offset: .init(width: 0, height: 2),
+                    radius: 2,
+                    color: .brand(.secondaryShadowColor),
+                    path: nil,
+                    corners: (.allCorners, 8))
```

Comment:
Unsure what I feel about a tuple here, maybe make a struct holding this information, like CornerStyle or something, and then you can do `.defaultAllCorners` or something :)

Reply:
True. I think the thing that's confusing is that radius here is actually not the radius but the spread of the shadow. maybe renaming that to something that makes more sense.